spec/ts-smoke/electron/renderer.ts

```diff
@@ -34,7 +34,7 @@ remote.getCurrentWindow().on('close', () => {
 })
 
 remote.getCurrentWindow().capturePage().then(buf => {
-  fs.writeFile('/tmp/screenshot.png', buf, err => {
+  fs.writeFile('/tmp/screenshot.png', (buf as any), err => {
```

Comment:
can we do better than `any`? What's the error here?

Reply:
Node.js v14 types this more strongly - we can do `buf as unknown as Buffer`perhaps

Reply:
<img width="734" alt="Screen Shot 2020-09-16 at 1 46 55 PM" src="https://user-images.githubusercontent.com/2036040/93385182-1b131280-f823-11ea-8700-7578c0e50de3.png">

Reply:
surely we shouldn't be trying to write a `NativeImage` to a file?